Sprint-2/errors/0.js

```diff
@@ -1,9 +1,15 @@
 // Predict and explain first...
 
 // call the function capitalise with a string input
-// interpret the error message and figure out why an error is occurring
+// interpret the error message and figure out why an error is occurring.
 
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+  str = `${str[0].toUpperCase()}${str.slice(1)}`;
```

Comment:
It would be clearer if you gave the capitalised string a new name. And therefore keep let to declare the new variable:
`let capString = ...`
`return capString`

Reply:
made changes and pushed.